configure_check_conditions.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>Phrasing Fixes:</p><p> </p><p>different kinds of default suites</p><p> </p><p>which come with pre-defined conditions</p><p> </p><p>or you may want to change the condition's parameters</p><p> </p><p>To remove a condition, start by printing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>